utils/client.js

```diff
@@ -2,9 +2,13 @@ import ApolloClient from 'apollo-boost';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 
 const PORT = process.env.PORT || 3000;
-const path = process.env.ENV ==='development'?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
+const path = process.env.ENV;
+const basePath =
+  path === 'development'
+    ? `http://localhost:${PORT}`
+    : 'https://vcars.herokuapp.com';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7128486)
```suggestion
    : "https://vcars.herokuapp.com";
```